docs/extend/extend-desktop/mvd-server-config.md

```diff
@@ -1,15 +1,24 @@
 The Zowe's App Server and ZSS rely on many required or optional parameters to run, which includes setting up networking, deployment directories, plugin locations, and more. 
```

Comment:
This article does not conform to Zowe doc article standards. There should be one a single H1 header at the beginning of the article. Currently, there are multiple H1 headers and the title appears as the URL: mvd-server-config.